ISL.Providers.Notifications.GovUkNotifyIntercept/Services/Foundations/Notifications/NotificationService.cs

```diff
@@ -62,5 +64,43 @@ public ValueTask<string> SendSmsAsync(
 
         public static dynamic GetValueOrNull(Dictionary<string, dynamic> dictionary, string key) =>
             dictionary.ContainsKey(key) ? dictionary[key] : null;
+
+        virtual internal async ValueTask<SubstituteInfo> SubstituteInfoAsync(
+            Dictionary<string, dynamic> personalisation)
+        {
+            var identifier = GetValueOrNull(personalisation, configurations.IdentifierKey);
```

Comment:
rename configurations to notifyConfigurations as it is not quite clear what it is without going to the top.  Change line 17,19, 22